### ISSUE TITLE
Affichage des données : pas d'accès à l'ensemble des données d'un secteur

### ISSUE THREAD

**Issue by okristo** [cbn-alpin/sinp-paca-tickets#9]
La limite des 50 000 dernières observations est maintenue même lorsqu’on zoome : les données affichées ne sont pas actualisées selon le niveau d'échelle. Il faudrait avoir les 50 000 dernières données du secteur affiché à l'écran et non pas de l'ensemble de PACA. Pas moyen d’afficher l’ensemble des données sur un périmètre restreint ?

**Comment by jpm-cbna** [comment]
Oui, c'est normal. La carte n'affiche que les observations correspondant à ta recherche avec les filtres dans la colonne de gauche.

Pour voir les observations d'un périmètre restreint 2 possibilités :
-  utiliser le filtre "Commune"
-  utiliser un des 3 icônes du haut à gauche sur la carte pour tracer un secteur sur la carte puis cliquer sur le bouton "Rechercher" de la colonne de gauche

Concernant la 2nd possibilité toutes idée visant à rendre plus intuitif ce fonctionnement est la bienvenue !

**Comment by okristo** [comment]
Besoin exprimé lors de la réunion technique Silene du 08/04/25 : interroger les données sur l'emprise de la carte via un bouton "Relancer la recherche sur l'emprise de la carte" (à l'image de ce qui est fait dans Simethis).

**Comment by jpm-cbna** [comment]
> Besoin exprimé lors de la réunion technique Silene du 08/04/25 : interroger les données sur l'emprise de la carte via un bouton "Relancer la recherche sur l'emprise de la carte" (à l'image de ce qui est fait dans Simethis).

Si je comprends bien, l'idée c'est de pouvoir limiter la recherche à la bbox correspondant à l'emprise de la carte que l'utilisateur visualise dans son navigateur quand il clique sur le bouton ?

A première vue, il me semble que ce bouton devrait:
- compléter le bouton "RECHERCHER" de la zone de filtre (2nd bouton ou flêche de sélection d'une option ?)
- être accessible seulement à un niveau de zoom donné (pour éviter de récupérer trop de données) => valeur configurable
